integration.md

```diff
@@ -200,9 +202,9 @@ every 1.day do
       origin_proof:  user.recurring_payment_reference
     )
 
-    authorization_response = purchase.authorize!
-    if authorization_response.succeed?
-      user.monthly_payments.create(Date.today, authorization_response)
+    success, response_data, response_code = purchase.order!
+    if success
+      user.monthly_payments.create(Date.today, response_data)
```

Comment:
Now that you've turned my gaze to this whole `user` business I really come to question this bit.
I guess this is better since it's more of a standalone task sort of thing (also there's no getting around it if you want to iterate on users, though we may want to change the `params[:userToken]` here to take advantage of the user objects this time around).